modules/app.xql

```diff
@@ -1068,86 +1065,24 @@ return <span> {$prev} | {$next}</span>
-                            var rootURL = ']]>{$config:nav-base}<![CDATA[';
-                            var postData =]]>{$visData}<![CDATA[;
-                            var id = ']]>{$id}<![CDATA[';
-                            var type = ']]>{$mode}<![CDATA[';
-                            $.get(dataURL, function(data) {
-                                     makeGraph(data,"850","400",rootURL,type);                
-                                }, "json"); 
-                            /*
-                            if($('#graphVis svg').length == 0){}
-                            jQuery(window).trigger('resize');
-                            */
-                        });
-                ]]></script>
-                <style><![CDATA[
-                    .d3jstooltip {
-                      background-color:white;
-                      border: 1px solid #ccc;
-                      border-radius: 6px;
-                      padding:.5em;
-                      font-size:10px; 
-                      }
-                    .nodelabel {font-size:12px; font-color: #666;}
-                    
-                    #legendContainer {
-               	    font-size: 12px; 
-               	    color: #666;
-                       position: absolute;
-                       top: 20px;
-                       right: 10px;
-                       background-color: white;
-                       width: 25%;
-                       min-height:200px;
-                       max-height:400px;
-                       padding: 4px;
-                       border-style: solid;
-                       border-radius: 4px;
-                       border-width: 1px;
-                       box-shadow: 3px 3px 10px rgba(0, 0, 0, .5);
-                       font-size:10px; 
-                       font-color: #666;
-                       overflow: auto;
-                    } 
-                    #legendContainer.legend h3 {font-size: 18px; text-align:center; margin: 8px;}
-                    #legendContainer.legend h4 {font-size: 14px; margin: 8px;}
-                    .filterList {margin-left: 16px;}
-                    #legendContainer button.filter {
-                      background: none!important;
-                      border: none;
-                      padding: 0!important;
-                      cursor: pointer;
-                    }
-                    ]]>
-                </style>
-                <script src="{$config:nav-base}/d3xquery/js/vis.js" type="text/javascript"/>
-        </div>      
+declare function app:data-visualization($node as node(), $model as map(*), $json-file as xs:string?, $mode as xs:string?, $locus as xs:string?, $relationship as xs:string?, $height as xs:string?, $width as xs:string?) {
+ d3xquery:data-visualization($model("hits"), $json-file, $mode, $locus, $relationship, $height, $width)
```

Comment:
@wsalesky I would guess the problem I'm getting with the d3xquery namespace is something to do with this?